awswrangler/s3/_read_parquet.py

```diff
@@ -246,14 +249,25 @@ def _read_parquet_file(
         s3_additional_kwargs=s3_additional_kwargs,
         s3_client=s3_client,
     ) as f:
-        pq_file: Optional[pyarrow.parquet.ParquetFile] = _pyarrow_parquet_file_wrapper(
-            source=f,
-            coerce_int96_timestamp_unit=coerce_int96_timestamp_unit,
-        )
-        if pq_file is None:
-            raise exceptions.InvalidFile(f"Invalid Parquet file: {path}")
+        if schema and version.parse(pa.__version__) >= version.parse("8.0.0"):
```

Comment:
We need to re-add `packaging` as a dependency :)

I removed it after we no longer needed it ([PR#2292](https://github.com/aws/aws-sdk-pandas/pull/2292/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711)).

Reply:
Ah right thanks